src/parser/mage/arcane/modules/features/ArcanePower.js

```diff
@@ -212,7 +245,7 @@ class ArcanePower extends Analyzer {
   suggestions(when) {
     when(this.cooldownSuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<>You cast <SpellLink id={SPELLS.ARCANE_POWER.id} /> {this.badUses} times without meeting the pre-requisites. Before casting Arcane Power you should ensure that you have 4 Arcane Charges{this.hasOverpowered ? '' : ' and at least 40% mana to avoid going OOM during Arcane Power'}. {this.hasRuneOfPower ? 'Additionally, you should ensure that you have at least 1 charge of Rune of Power, and you should cast it immediately before casting Arcane Power.' : ''}</>)
+        return suggest(<>You cast <SpellLink id={SPELLS.ARCANE_POWER.id} /> with less than 4 Arcane Charges {this.lowChargesCast} times and with low mana {this.lowManaCast} times. {this.hasRuneOfPower ? 'Additionally, you cast Arcane Power without Rune of Power ' + this.noRuneCast + ' times and delayed Arcane Power after Rune of Power ' + this.delayedRuneCast + ' times.' : ''} It is very important that you do not cast Arcane Power unless you have 4 Arcane Charges, have over 40% Mana (Unless talented into <SpellLink id={SPELLS.OVERPOWERED_TALENT.id} />), and <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id} /> was cast immediately before Arcane Power (if talented into Rune of Power).</>)
```

Comment:
Maybe something similar to the statistic box should be put into the suggestion box? Like:

You cast Arcane Power sub-optimally X of Y times.
- You should have 4 Arcane Charges when you cast Arcane Power. X casts without 4 charges.
- You should cast Rune of Power directly before Arcane Power. X casts without an optimal Rune of Power.
- You should have enough mana (~X% without Overpowered) to not run OOM during the Arcane Power. X casts with low mana.

Reply:
i wanted to do something like that, but couldnt figure out how to get an ordered list in there .... ill have to play with it some more.